tests/pages/add-address-page.tsx

```diff
@@ -0,0 +1,52 @@
+import userEvent from "@testing-library/user-event";
```

Comment:
You forgot to add `.test` in your name file :).

Reply:
🤦